### ISSUE TITLE
black market of nope

### ISSUE THREAD

**Issue by hexagonrecursion** [naikari/naikari#11]
#### Naikari version

v0.6.0

#### Describe the bug

Some planets (e.g. Alleyway) have a black market despite not selling ships or outfits. Is this intended? What is the point?

https://github.com/naikari/naikari/blob/af8165c0a0c78d3e634551fbb7bd5456d9df6eed/dat/assets/alleyway.xml#L24

#### Expected behavior

Remove the black market or add ships or outfits

**Comment by onpon4** [comment]
Yes, it is intended, the reason is that these planets will offer illegal missions to you (currently just the ship stealing mission, which is found at the bar). We intend to extend this further by adding some kind of event selling sketchy or otherwise unique ships and outfits through NPCs on black market planets (likely functioning similar to the ship stealing mission, where you find NPCs at black market locations labeled as "sketchy salesperson" or similar and they offer to sell you a particular randomized item).

🕷️
